src/ProjectTemplates/Web.ProjectTemplates/content/RazorPagesWeb-CSharp/.template.config/dotnetcli.host.json

```diff
@@ -71,6 +71,18 @@
     "RazorRuntimeCompilation": {
```

Comment:
@jmprieur can you review the list of arguments and see if they are applicable to Microsoft.Identity.Web? Specifically, we used to specify the following for OrgAuth and IndividualB2CAuth scenarios:

```
    "AADInstance": {
      "longName": "aad-instance",
      "shortName": ""
    },
    "AAdB2CInstance": {
      "longName": "aad-b2c-instance",
      "shortName": ""
    },
    "SignUpSignInPolicyId": {
      "longName": "susi-policy-id",
      "shortName": "ssp"
    },
    "ResetPasswordPolicyId": {
      "longName": "reset-password-policy-id",
      "shortName": "rp"
    },
    "EditProfilePolicyId": {
      "longName": "edit-profile-policy-id",
      "shortName": "ep"
    },
    "OrgReadAccess": {
      "longName": "org-read-access",
      "shortName": "r"
    },
    "ClientId": {
      "longName": "client-id",
      "shortName": ""
    },
    "CallbackPath": {
      "longName": "callback-path",
      "shortName": ""
    },
    "Domain": {
      "longName": "domain",
      "shortName": ""
    },
    "TenantId": {
      "longName": "tenant-id",
      "shortName": ""
    },
```

Taking a quick look through the code in Microsoft.Identity.Web, it seems like these are unused and should be removed:
```
AADInstance
AAdB2CInstance
OrgReadAccess
```
and the rest are still relevant. Is that correct?